plugins/flatpak/gs-plugin-flatpak.c

```diff
@@ -1825,7 +1825,7 @@ gs_plugin_flatpak_update (GsPlugin *plugin,
 	}
 
 	/* build and run transaction */
-	transaction = _build_transaction (plugin, flatpak, interactive, cancellable, error);
+	transaction = _build_transaction (plugin, flatpak, GS_FLATPAK_ERROR_MODE_STOP_ON_FIRST_ERROR, interactive, cancellable, error);
```

Comment:
I think this would be a regression. We really want Software to apply as many updates as it can, even if some of them fail.

Otherwise, you can get into a state where you have a vast number of app updates pending; but the first one happens to always fail, and so you never get updates. But each time, Software will download them all before applying them, wasting bandwidth and disk space. Particularly when the reason that some of the updates fail is “you don't have enough disk space” you really want to apply some rather than none, because updating apps temporarily uses additional disk space.

https://gitlab.gnome.org/GNOME/gnome-software/-/issues/1929